src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import bReducer from './books/books';
+import cReducer from './categories/categories';
+
+const reducer = combineReducers({
+  bReducer,
+  cReducer,
+});
+
+const store = configureStore({
+  reducer,
+});
```

Comment:
- [ ] @TechnoAndy great work in your store, Kindly ensure you pass the combined reducer  (reducer) to your configure store method, currently it's not, [ tutorial on how to do it ](https://redux-toolkit.js.org/usage/usage-guide), alternative you could pass the reducers direct to configure store method:
eg `const rootReducer = combineReducers({
  books: bookReducer,
  categories: categoriesReducer,
});
`